### ISSUE TITLE
Support hyphenated IP ranges

### ISSUE THREAD

**Issue by william-tran** [cloudfoundry-incubator/asg-creator#4]
Some specifications for IPs to include and exclude could be in the form of hyphenated ranges, eg:

```
included_networks:
- 10.68.192.0/20

excluded_ranges:
- 10.68.192.1-10.68.196.126
- 10.68.196.251-10.68.207.255
```

New properties `included_ranges` and `excluded_ranges` could be added, or the usage could be overloaded into `included_networks` and `excluded_networks`.

**Comment by krishicks** [comment]
Sounds good to me.

**Comment by krishicks** [comment]
Fixed in 0.2.0. Note the backwards-incompatible changes: `included_*` and `excluded_*` are now just `include` and `exclude`.
